uniflow/flow/config.py

```diff
@@ -186,23 +186,22 @@ def check_labels(self) -> Dict[str, list]:
 
 
 @dataclass
-class RaterClassificationOpenAIGPT4Config(RaterConfig):
+class RaterForClassificationOpenAIGPT4Config(RaterConfig):
     """
-    The configuration primarily focuses on setting up the parameters for utilizing GPT-4 to evaluate the 
+    The configuration primarily focuses on setting up the parameters for utilizing GPT-4 to evaluate the
     correctness of answers in relation to given questions and contexts.
-    
+
     Attributes:
         flow_name (str): Name of the rating flow, default is "RaterFlow".
-        model_config (ModelConfig): Configuration for the GPT-4 model. Includes model name ("gpt-4"),
-                                    the server ("OpenAIModelServer"), number of calls (1), temperature (0.2),
+        model_config (ModelConfig): Configuration for the GPT-4 model. Includes model name ("gpt-4"),\
+                                    the server ("OpenAIModelServer"), number of calls (1), temperature (0.2),\
                                     and the response format (plain text).
         label2score (Dict[str, float]): Mapping of labels to scores, default is {"Yes": 1.0, "No": 0.0}.
-        guided_prompt_template (GuidedPrompt): Template for guided prompts used in rating. Includes instructions
-                                               for rating, along with examples that detail the context, question,
+        guided_prompt_template (GuidedPrompt): Template for guided prompts used in rating. Includes instructions\
+                                               for rating, along with examples that detail the context, question,\
                                                answer, label, and explanation for each case.
     """
 
-    flow_name: str = "RaterFlow"
     model_config: ModelConfig = field(
```

Comment:
according to empirical result, for a LLM rater task, reproducible result with deterministic `temperature=0` perform the closely to human rater, so please set `temperature=0` as default value.

Reply:
I will set this default value. But I remember `temperature = 0` just mean do no scaling when softmax to get token probabilities. Decoding can still be a sampling process (non-deterministic). See https://txt.cohere.com/llm-parameters-best-outputs-language-ai/

Reply:
Yes, because most user are cost sensitive, so we will set to the most greedy approach as default. However, through changing temperate and number of sample, we can easily re-config to CoT@n for chain of thoughts with self consistency.